src/rules/builtins.rs

```diff
@@ -266,19 +258,15 @@ mod tests {
         assert!(result.is_ok(), "{:#?}", result);
 
         let locations = result.unwrap();
-        assert_eq!(locations.len(), 1);
-        assert_eq!(locations[0].range, TextRange::new(16.into(), 19.into()));
-        assert_eq!(&root.syntax().to_string()[locations[0].range], "NVL");
+        check_locations(&locations, r#"[RuleMatch(16..19, "NVL")]"#);
 
-        assert_eq!(location, TextRange::new(7.into(), 15.into()));
-        assert_eq!(&root.syntax().to_string()[location], "coalesce");
+        check_applied_location(&root, location, 7..15, "coalesce");
```

Comment:
Same here, this should be already covered by the text comparison using `check_node`.